server/index.js

```diff
@@ -1,11 +1,14 @@
-const newrelic = require('newrelic');
```

Comment:
Why did you take out this line and just have "require('newrelic')? Was this an error?

Reply:
They both work but newrelic does not need to be declared as a variable.